### PR TITLE
[Bug 1169416] added funtionality to show full locale name in show_tra…

### PR DESCRIPTION
**Pull request by skiee-mahir**
…nslations.html

### CODE REVIEW

**Comment by skiee-mahir** [comment]
@mythmon can you check?

File: kitsune/wiki/templates/wiki/show_translations.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% extends "wiki/base.html" %}
+k{% extends "wiki/base.html" %}
```
Comment (safwanrahman) [inline kitsune/wiki/templates/wiki/show_translations.html]: k? why?

File: kitsune/wiki/views.py [hunk 2]
```diff
@@ -1382,13 +1382,17 @@ def show_translations(request, document_slug):
     translated_locales = []
     untranslated_locales = []
 
-    translated_locales.append(document.locale)
-    translated_locales.extend(document.translations.all().values_list(
-        'locale', flat=True))
+    # Makes Sure "English (en-US)" is always on top
+    translated_locales.append((document.locale, LOCALES[document.locale].native))
+    # Gets all the translated locale codes
+    translated_locales_code = document.translations.all().values_list(
```
Comment (safwanrahman) [inline kitsune/wiki/views.py]: hmm! Don't call '''all()''' unless you don't neeed to make database call everytime.

**Comment by safwanrahman** [comment]
Just some simple neat and clean up.
Otherwise it seems ok. 
@mythmon I think this should get r+

**Comment by skiee-mahir** [comment]
@safwanrahman do you have any idea why the test failed? When it was OK with the mistaken k?

**Comment by safwanrahman** [comment]
@skiee-mahir Seems to be a problem with Travis. Can yo fix the above mentioned issue and squash into one commit and push again?

File: kitsune/wiki/views.py (hunk 2 above)
Comment (mythmon) [inline kitsune/wiki/views.py]: `all()` here doesn't actually do anything. It is only useful when there are no other method calls after `.objects`. Please remove it.

Comment (mythmon) [inline kitsune/wiki/views.py]: Oh, I see that this code already exists. I'll fix it later then.

**Comment by mythmon** [comment]
r+

**Pull request merged**